network/wsPeer_test.go

```diff
@@ -270,7 +270,7 @@ func getProtocolTags(t *testing.T) []string {
 		for _, spec := range genDecl.Specs {
 			if valueSpec, ok := spec.(*ast.ValueSpec); ok {
 				for _, n := range valueSpec.Names {
-					if n.Name == "TagLength" {
+					if strings.HasSuffix(n.Name, "MaxSize") || n.Name == "TagLength" {
 						continue
 					}
 					declaredTags = append(declaredTags, n.Name)
```

Comment:
if I'm reading this block right, we're trying to iterate over the valueSpec of a given tag, and through excluding MaxSize and TagLength, we expect that the only remaining thing is the declared tag's name? This seems fragile if/when more is added to the spec. Since it's just a test I don't know that it's worth pursuing.

Reply:
Yeah it is fragile as you can see since both myself and Shant had to add exclusions here for MaxSize and TagLength respectively. Hopefully we won't be adding a lot more specs to protocol.Tags